lib/watir/elements/element.rb

```diff
@@ -395,13 +376,13 @@ def wd
 
     #
     # Returns true if this element is visible on the page.
+    # Raises exception if element does not exist
     #
     # @return [Boolean]
     #
 
     def visible?
-      assert_exists
-      element_call { @element.displayed? }
+      element_call(:assert_exists) { @element.displayed? }
```

Comment:
Hmmm... Because `visible?` throws an exception if it doesn't exist, and `present?` returns false if it doesn't exist... Interesting idea. A few thoughts in no particular order

1. Most people use these methods interchangeably right now
2. Clarifying the meaning of each might even be harder than before. Is `element.visible?` is now equivalent to `element.wait_until(&:exists?).present?` easy enough to understand?
3. Is this a pattern that is likely to be used?
4. I don't like having two methods that are mostly used for the same purpose like we have now

I actually think I like this, but I'm wondering if this shouldn't be made its own thing with its own blog post, etc. At the very least I'd like to get more opinions on it.